src/proxy-ui-api/frontend/src/views/KeysAndCertificates/ApiKey/ApiKey.vue

```diff
@@ -105,28 +227,91 @@ export default Vue.extend({
           .catch((error) => this.$store.dispatch('showError', error));
       }
     },
+    editKey(apiKey: ApiKey): void {
+      this.selectedKey = apiKey;
+      this.selectedRoles = [...this.selectedKey.roles];
+      this.showEditDialog = true;
+    },
+    showRevokeDialog(apiKey: ApiKey): void {
+      this.selectedKey = apiKey;
+      this.confirmRevoke = true;
+    },
     createApiKey(): void {
       this.$router.push({
         name: RouteName.CreateApiKey,
       });
     },
+    translateRoles(roles: string[]): string[] {
+      return !roles
+        ? []
+        : roles.map((role) => this.$t(`apiKey.role.${role}`) as string);
+    },
+    async revokeApiKey() {
```

Comment:
Same `async` keywords here. Prolly not needed

Reply:
Fixed